test/push.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source $(dirname "${BASH_SOURCE[0]}")/utils.sh
+
+# DEPLOY_BUCKET is the gcs bucket where the tarball should be stored in.
+DEPLOY_BUCKET=${DEPLOY_BUCKET:-"cri-containerd-staging"}
+# DEPLOY_DIR is the directory in the gcs bucket to store the tarball.
+DEPLOY_DIR=${DEPLOY_DIR:-""}
+# BUILD_DIR is the directory of the bulid out.
+BUILD_DIR=${BUILD_DIR:-"_output"}
+# TARBALL is the tarball name.
+TARBALL=${TARBALL:-"cri-containerd.tar.gz"}
+# LATEST is the name of the latest version file.
+LATEST=${LATEST:-"latest"}
```

Comment:
do we want the suffix `.txt`? does it render fine in a browser (not sure if it will make a diff)

Reply:
I am not really sure how these files are used, this was one thing I couldn't track down in the existing setups. I am just leaving them alone, but cleaning it up after it is working would be great. It looks like we will have to pull in our own e2e configuration anyway. I'm not sure if we can control the configuration script from this repo though.